VARIABLES.md

```diff
@@ -140,11 +140,11 @@ Default:  true
 
 ***
 
-### disk_ratio
+### free_disk_space
```

Comment:
Would `required_disk_space`, or perhaps even `required_disk_space_mb` make this variable more understandable without refering to the docs?

Reply:
Agreed, at least a unit postfix should be included in the name.